modules/git/repo_tag.go

```diff
@@ -121,7 +121,9 @@ func (repo *Repository) GetTagInfos(page, pageSize int) ([]*Tag, int, error) {
 	rc := &RunOpts{Dir: repo.Path, Stdout: stdoutWriter, Stderr: &stderr}
 
 	go func() {
-		err := NewCommand(repo.Ctx, "for-each-ref", CmdArg("--format="+forEachRefFmt.Flag()), "--sort", "-*creatordate", "refs/tags").Run(rc)
+		err := NewCommand(repo.Ctx, "for-each-ref").
+			AddOptionFormat("--format=%s", forEachRefFmt.Flag()).
+			AddArguments("--sort", "-*creatordate", "refs/tags").Run(rc)
```

Comment:
```suggestion
		err := NewCommand(repo.Ctx, "for-each-ref", "--sort", "-*creatordate", "refs/tags").
			AddOptionFormat("--format=%s", forEachRefFmt.Flag()).Run(rc)
```

Reply:
It just came from old code. I won't touch old logic too much. To make sure the code is clear enough during reviewing.